### PR TITLE
Fixes to support new composer backend

### PR DESCRIPTION
**Pull request by kaviththiranga**
Composer's backend modules were refactored and these changes are to support those changes

### CODE REVIEW

File: modules/ballerina-tools/conf/composer-config.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#port: 9093
```
Comment (hevayo) [inline modules/ballerina-tools/conf/composer-config.yml]: Shall we remove the file or content if not used ?

**Review by hevayo — APPROVED** [review]
Approved with a comment

File: modules/ballerina-tools/conf/composer-config.yml (hunk 1 above)
Comment (kaviththiranga) [reply modules/ballerina-tools/conf/composer-config.yml]: @hevayo Used this as sample config, shall we add this to a readme then?

Comment (kaviththiranga) [reply modules/ballerina-tools/conf/composer-config.yml]: @hevayo added a new line to top explaining the structure

**Pull request merged**